<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addEntry</class>
 <widget class="QDialog" name="addEntry">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>160</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Edit Entry</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(238, 238, 236);
border-color: rgb(136, 138, 133);</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="timeLabel_2">
     <property name="text">
      <string>Entry Time</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="summLabel">
     <property name="text">
      <string>Summ of Operation</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineSumm">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelComment">
     <property name="text">
      <string>comment</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEditComment">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="Side">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Side of operation&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>+</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="confirmChanges">
       <property name="text">
        <string>Add Entry</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
